### ISSUE TITLE
the loss cannot decline,and accuracy always is zero

### ISSUE THREAD

**Issue by Levi-zhan** [yihui-he/caffe-pro#3]
hello,when I use your caffe-pro-master and ResNet-50-test.prototxt to train ,the loss cannot decline,and accuracy always is zero.But in your log,the prototxt you used is resnet_50.prototxt,why?
this is the log I trained:
I0226 09:02:42.233305 14020 solver.cpp:239] Iteration 44000 (1.20803 iter/s, 1655.59s/2000 iters), loss = 6.90792
I0226 09:02:42.233350 14020 solver.cpp:258]     Train net output #0: accuracy@1 = 0
I0226 09:02:42.233361 14020 solver.cpp:258]     Train net output #1: accuracy@5 = 0
I0226 09:02:42.233371 14020 solver.cpp:258]     Train net output #2: loss = 6.90991 (* 1 = 6.90991 loss)
I0226 09:02:42.233382 14020 sgd_solver.cpp:112] Iteration 44000, lr = 0.1
you can see,it have run 44000 iteration,but accuracy is zero .
Please give me some adivse ,thank you

**Comment by ethanhe42** [comment]
Can you train it successfully with `resnet_50.prototxt`? 
If no, check whether you can train a caffe provided alexnet.

**Comment by Levi-zhan** [comment]
I'm very to receiver your reply，I did retrain use resnet_50.prortotxt，and it successful ，the log is very close to yours

**Comment by Levi-zhan** [comment]
why closed，it's very important for me，help me，please

**Comment by ethanhe42** [comment]
I don't know the solution. The ImageData layer works well in other codes.
